style2.css

```diff
@@ -60,7 +66,7 @@
     font-family: "Poppins", sans-serif;
     font-style: normal;
     font-weight: 700;
-    font-size: 50px;
+    font-size: 49px;
     line-height: 52px;
   }
```

Comment:
- [ ] The JavaScript should be set correctly in project repository. => Looking at the linters check section of your PR, I couldn't see where HTML, Style, and Javascript linters are being shown, kindly make sure that they are set correctly and running before you submit another code review request
- [ ] The popup should be responsive, it should fit mobile and desktop sizes properly. => your version doesn't fit in mbile screens at all, kindly address this issue by implementing the responsiveness of the popup window ; When your page is browsed from a screen of 320px of width, its showing an horizontal scroll which wasn't supposed to be there, kindly make sure that scroll is disabled and the page fits properly on small screens starting from 320px of width.

https://user-images.githubusercontent.com/56886064/145959689-6b4dba98-06fb-4897-bd4b-99124db39b13.mp4